pkg/tests/ossm/smoke_test.go

```diff
@@ -183,18 +182,28 @@ func assertSMCPDeploysAndIsReady(t test.TestHelper, ver version.Version) {
 	oc.WaitSMCPReady(t, meshNamespace, smcpName)
 }
 
-// func assertUninstallDeletesAllResources(t test.TestHelper, ver version.Version) {
-// 	t.LogStep("Delete SMCP in namespace " + meshNamespace)
-// 	oc.DeleteFromString(t, meshNamespace, GetSMMRTemplate())
-// 	DeleteSMCPVersion(t, meshNamespace, ver)
-// 	retry.UntilSuccess(t, func(t TestHelper) {
-// 		oc.GetAllResources(t,
-// 			meshNamespace,
-// 			assert.OutputContains("No resources found in",
-// 				"All resources deleted from namespace",
-// 				"Still waiting for resources to be deleted from namespace"))
-// 	})
-// }
+func assertUninstallDeletesAllResources(t test.TestHelper, ver version.Version) {
+	t.LogStep("Delete SMCP in namespace " + meshNamespace)
+	oc.DeleteFromString(t, meshNamespace, GetSMMRTemplate())
+	DeleteSMCPVersion(t, meshNamespace, ver)
+	retry.UntilSuccess(t, func(t TestHelper) {
+		shell.Execute(t,
+			fmt.Sprintf("oc get smcp -n %s", meshNamespace),
+			assert.OutputContains("No resources found in",
+				"SMCP is deleted from namespace",
+				"Still waiting for smcp to be deleted from namespace"))
+		shell.Execute(t,
+			fmt.Sprintf("oc get pods -n %s", meshNamespace),
+			assert.OutputContains("No resources found in",
+				"Pods are deleted from namespace",
+				"Still waiting for pods to be deleted from namespace"))
+		shell.Execute(t,
+			fmt.Sprintf("oc get services -n %s", meshNamespace),
+			assert.OutputContains("No resources found in",
+				"Services are deleted from namespace",
+				"Still waiting for services to be deleted from namespace"))
```

Comment:
Why not use `oc.Get`? If we have a high-level function, we should use it.

Also, you can get all three in a single command (`get smcp,pods,svc,`)

Reply:
Changes done, test working:
```
--- PASS: TestSmoke (170.71s)
    --- PASS: TestSmoke/install_bookinfo_with_smcp_v2.3 (47.88s)
    --- PASS: TestSmoke/upgrade_v2.3_to_v2.4 (44.56s)
    --- PASS: TestSmoke/delete_smcp_v2.4 (8.21s)
```